### PR TITLE
Truenth new lived experience page

### PR DESCRIPTION
**Pull request by achen2401**
address story:  
https://www.pivotaltracker.com/story/show/143547263

fixed issues with image distortion and quality of images

this PR can now be merged into the develop branch upon review.

### CODE REVIEW

File: portal/views/portal.py [hunk 1]
```diff
@@ -144,6 +144,18 @@ def exercise_and_diet():
 def lived_experience():
     return render_template('gil/lived-experience.html', user=current_user())
 
+@portal.route('/stories/david-andrew-story')
+def david_andrew_story():
```
Comment (ivan-c) [inline portal/views/portal.py]: These could probably all be served by a single view function, which should reduce some code duplication and should simplify the creation of language-specific versions of each GIL page (eg `gil/lived-experience_en_AU.html`).

There's a [good example here](http://flask.pocoo.org/snippets/126/), but let me know if you need any help :)

Comment (ivan-c) [inline portal/views/portal.py]: Actually- it's probably better to open a new PR for refactoring **all** of them to use a single view- maybe just try it out for the new views/templates ones you've added now?

**Comment by achen2401** [comment]
@ivan-c 
thank you for the great suggestion. yeah, I should create a new PR to refactor all to use one single view in the near future, but for the time being, I did consolidate the stories into one single view function.

**Comment by ivan-c** [comment]
Thanks for making those changes, Justin said he doesn't have any changes; going to merge

**Pull request merged**